client/state/sites/vouchers/actions.js

```diff
@@ -75,6 +79,68 @@ export const vouchersRequestFailureAction = ( siteId, error ) => {
 	};
 };
 
+export const vouchersAssignReceiveAction = ( siteId, serviceType, voucher ) => {
+	return {
+		type: SITE_VOUCHERS_ASSIGN_RECEIVE,
+		siteId,
+		serviceType,
+		voucher
+	};
+};
+
+/**
+ * Action creator function
+ *
+ * Return SITE_VOUCHERS_ASSIGN_REQUEST action object
+ *
+ * @param {Number} siteId - side identifier
+ * @param {String} serviceType - service type
```

Comment:
Similar note about the comment here - it's already obvious that this is the `serviceType`, so maybe it either needs no comment or a descriptive note, such as `could be 'Google' or 'Bing' etc…``